engine/image-loader/image-loader.lisp

```diff
@@ -26,5 +26,5 @@
 (defun load-image (path)
   "Takes path to an image and returns `image` structure.
    If the image format is unsupported throws error."
-  (ccase (intern (pathname-type path))
-    ('|tga| (lbge.image-loader.tga:tga path))))
+  (switch ((pathname-type path) :test 'equal)
```

Comment:
Better to be most specific and use `#'string=` here, since we're comparing strings.